Fix file stacking

mammo0 opened the pull request:
As discussed with @Shadowghost in #8004, there is a stacking rule (`VideoFileStackingRules`) in the file `Emby.Naming/Common/NamingOptions.cs` that makes no sense. It leads to a behavior that is not documented.

For example the files

- `300a.mkv`,
- `300b.mkv` and
- `300c.mkv`

will be treated as parts of the same movie. But according to the Shows documentation (https://jellyfin.org/docs/general/server/media/shows/#episodes-split-across-multiple-parts) _- don't know why this isn't also documented in the movie section -_ a `<parttype>` is required.
So normally the files should to be named like this:

- `300 parta.mkv`,
- `300 partb.mkv` and
- `300 partc.mkv`

I removed the corresponding rule and adjusted the affected test cases. Now the above naming (`300a`, ...) does no longer work. Instead the files have to be named like `300 parta`, etc. as it is documented.

Shadowghost commented on tests/Jellyfin.Naming.Tests/Video/VideoListResolverTests.cs:
```diff
@@ -332,7 +332,9 @@ public void TestFourSisters()
                 _namingOptions).ToList();
 
-            Assert.Single(result);
+            // The result should contain to individual movies
```
```suggestion
            // The result should contain two individual movies
```

mammo0 commented:
> I added the stacking documentation to shows [...]

You mean, you've now added it to the movies? Because the Shows documentation page had already that part.

Shadowghost commented:
He was the person who originally added the documentation.

We might want to move the stacking an versioning dosumentation to its own page though once the episdoe versioning PR is in.
